### PR TITLE
FriendMe Team Add Request

### PR DESCRIPTION
**Pull request by Solarleaf**
All team members present

### CODE REVIEW

**Comment by kbuffardi** [comment]
Your team file name looks good.

Please update this PR by making a commit to your fork's `main` that edits `README.md` by linking your team's name to your team's markdown file. See the previous semesters' teams for examples.

**Comment by Solarleaf** [comment]
Adding FriendMe link to readme.md

**Comment by Solarleaf** [comment]
Updated for relative path

File: teams/FriendMe.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#Developers
+* Leif Hasle, [Leif](https://github.com/Solarleaf)
```
Comment (kbuffardi) [inline teams/FriendMe.md:2]: In the square brackets, put the username

Comment (Solarleaf) [reply teams/FriendMe.md]: Should be updated

**Pull request merged**